packages/vue-query/src/useQueries.ts

```diff
@@ -214,7 +214,10 @@ export function useQueries<
         defaultedQueries.value,
         options as QueriesObserverOptions<TCombinedResult>,
       )
-      state.value = observer.getCurrentResult()
+      const [, getCombinedResultPersisted] = observer.getOptimisticResult(
+        defaultedQueries.value,
+      )
+      state.value = getCombinedResultPersisted()
```

Comment:
@DamianOsipiuk can you have look here please?

Reply:
Looks good 👍

Reply:
Although this test relies on `useQueries`, which has different mechanism of data retrieval.
I wonder if this would still work for regular `useQuery`. Need to add a test for that since there we rely on `getCurrentResult` which appears to have changed behavior.

Reply:
so - can we ship this or should we wait?

Reply:
I would rather add the test first.

Reply:
Sorry guys I was on vacation this week and had too much family work to do all of a sudden .. Please let me know if there is anything I can do

Reply:
@TkDodo it looks good from my side.